oauthenticator/oauth2.py

```diff
@@ -579,6 +588,27 @@ def build_access_tokens_request_params(self, handler, data=None):
 
         return params
 
+    def build_refresh_token_request_params(self, refresh_token):
+        """
+        Builds the parameters that should be passed to the URL request
+        that renew Access Token from Refresh Token.
+        Called by the :meth:`oauthenticator.OAuthenticator.refresh_user`.
+        """
+        params = {
+            "grant_type": "refresh_token",
+            "refresh_token": refresh_token,
```

Comment:
need to include 'scope' here too - the same one used on initial login, to ensure you get back an equivalent id_token/access_token

Reply:
apologies, looks like this is only required on the original authorization code request. the scopes for the token_info and refresh_token requests do not affect the responses